calico/getting-started/kubernetes/quickstart.mdx

```diff
@@ -1,170 +1,500 @@
+We'll do this by applying a Global Calico Network Policy that denies all ingress and egress traffic by default.
+
+In this step, you will:
+* **Implement a global default deny policy:** Use a Global Calico Network Policy to deny all ingress and egress traffic by default.
+* **Verify access is denied:** Use your BusyBox pod to confirm that the policy is working as expected.
+
+1. Create a Global Calico Network Policy to deny all traffic except for the necessary system namespaces:
+
+   ```bash
+   kubectl create -f - <<EOF
+   apiVersion: projectcalico.org/v3
+   kind: GlobalNetworkPolicy
+   metadata:
+     name: default-deny
+   spec:
+     selector: projectcalico.org/namespace not in {'kube-system', 'calico-system', 'calico-apiserver'}
+     types:
+       - Ingress
+       - Egress
+   EOF
    ```
-   node/<your-hostname> untainted
+   ```bash title="Expected output"
+   globalnetworkpolicy.projectcalico.org/default-deny created
    ```
 
-1. Confirm that you now have a node in your cluster with the following command.
+1. Now go back to your BusyBox shell and test access to the NGINX server again:
 
+   ```bash
+   wget -qO- http://nginx
    ```
-   kubectl get nodes -o wide
+
+   You should see the following output, indicating that access is denied:
+
+   ```bash title="Expected output"
+   wget: bad address 'nginx'
    ```
 
-   It should return something like the following.
+1. Test access to the public internet again:
 
+   ```bash
+   wget -qO- https://docs.tigera.io/pod-connection-test.txt
    ```
-   NAME              STATUS   ROLES    AGE   VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION    CONTAINER-RUNTIME
-   <your-hostname>   Ready    master   52m   v1.12.2   10.128.0.28   <none>        Ubuntu 18.04.1 LTS   4.15.0-1023-gcp   docker://18.6.1
+
+   You should see the following output, indicating that egress traffic is also denied:
+
+   ```bash title="Expected output"
+   wget: bad address 'docs.tigera.io'
    ```
 
-Congratulations! You now have a single-host Kubernetes cluster with $[prodname].
+1. Return to your browser to see the denied flow logs appear in the Whisker web console.
+   You should see two denied flows to `coredns`.
+
+   <Screenshot src="/img/quickstart-whisker3.png" alt="whisker3" />
+   *Figure {figCount++}: Whisker web console showing denied flows to NGINX server and `https://docs.tigera.io`.*
+
+
+   By following these steps, you have successfully implemented a global default deny policy and verified that it is working as expected.
+
+## Step 6. Create targeted network policy for allowed traffic
+
+Now that you have a default deny policy in place, you need to create specific policies to allow only the necessary traffic for your applications to function.
+The `default-deny` policy blocks all ingress and egress traffic for pods not in system namespaces, including our `access` (BusyBox) and `nginx` pods in the `quickstart` namespace.
```

Comment:
Given that you've shown that DNS lookups are being blocked in Whisker, should you not talk about how to use the Whisker record to allow the blocked DNS traffic?

And then you'll see that the wget fails because of ingress policy on the default deny, and you can use that information to create the next policy.

Reply:
That's a great idea. I'll see about making that change in the next revision. Thanks!

Reply:
I'll track this idea separately, along with a similar one in the CC version.